src/main/java/org/kaiaccount/account/eco/commands/bank/DeletePlayerBankAccountCommand.java

```diff
@@ -0,0 +1,56 @@
+package org.kaiaccount.account.eco.commands.bank;
+
+import org.bukkit.OfflinePlayer;
+import org.jetbrains.annotations.NotNull;
+import org.kaiaccount.AccountInterface;
+import org.kaiaccount.account.eco.message.Messages;
+import org.kaiaccount.account.eco.message.messages.error.SourceOnlyCommandMessage;
+import org.kaiaccount.account.inter.type.named.bank.player.PlayerBankAccount;
+import org.kaiaccount.account.inter.type.player.PlayerAccount;
+import org.mose.command.ArgumentCommand;
+import org.mose.command.CommandArgument;
+import org.mose.command.arguments.operation.ExactArgument;
+import org.mose.command.arguments.simple.text.StringArgument;
+import org.mose.command.context.CommandContext;
+
+import java.util.List;
+import java.util.Optional;
+
+public class DeletePlayerBankAccountCommand implements ArgumentCommand {
+
+    public static final ExactArgument CREATE = new ExactArgument("close");
+    public static final StringArgument NAME = new StringArgument("name");
```

Comment:
There is a PlayerBankAccount argument 
https://github.com/EconToolBox/EcoTools/blob/main/src/main/java/org/kaiaccount/account/eco/commands/argument/account/PlayerBankArgument.java

This enforces the user to enter a bank account. I would recommend using the static methods on it and combine it with the ``PemissionOrArgument`` which changes the context of the argument depending on if the sender of the command has the provided command.

I would end up being something like


```java
new PermissionOrArgument("name", (sender) -> sender.hasPermission(Permissions.DELETE_BANK_OTHER, PlayerBankArgument.allPlayerBanks(""), PlayerBankArgument.senderBanks("")));
```

so that if the sender has the permission, all player banks argument is used, however if the player doesnt have the permission, the sender banks argument will be used